code/game/objects/effects/anomalies.dm

```diff
@@ -184,6 +184,10 @@
 /obj/effect/anomaly/grav/high/proc/setup_grav_field()
 	grav_field = new(src, 7, TRUE, rand(0, 3))
 
+/obj/effect/anomaly/grav/high/detonate()
+	for(var/obj/machinery/gravity_generator/main/the_generator in GLOB.machines)
+		the_generator.blackout()
```

Comment:
I thought it would be funnier if it just disabled every generator on the "sector" but I changed it to not be annoying for spawners to have to stop to reboot their generators.